### ISSUE TITLE
Implement additional output results for Susi AI

### ISSUE THREAD

**Issue by mariobehling** [fossasia/labs.fossasia.org#188]
Apart from text answers Susi AI is already able to provide maps and search results. Please propose and implement a list of additional services and media in the Susi AI server, that can be called by the API.

**Comment by ghost-60** [comment]
Maybe play a media file or any open any application that is on the PC. For example, If I write "Hey I would like to edit this picture" it could automatically open any photo editing software that is there and say "You could try it here" or else "you need to have _this_ installed to do that" and suggest other options.

**Comment by ansh103** [comment]
Is this issue still open? I saw a number of projects mentioning they aim to do so. Yet when I do ask SUSI for an image. It doesn't provide me with one. Clearly I asked for a pizza image and didn't get one. But it's able to give text replies. Also there are no replies on general questions like "Tell me a joke"  or "Recite me a poem". Such additional output requests should have been solved. So is it still open? 
![img_20180130_161210_676](https://user-images.githubusercontent.com/31566246/35562127-7052a5fc-05d8-11e8-83e4-a2e298ab8a8f.JPG)
